src/common.jl

```diff
@@ -80,6 +74,33 @@ function solve{uType,tType,isinplace,algType<:ODEJLAlgorithm,F}(prob::AbstractOD
                           minstep=dtmin,
                           initstep=dt,
                           points=points)
+    elseif typeof(alg) <: ode4
+        ts,timeseries_tmp = ODE.ode4(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4ms
+        ts,timeseries_tmp = ODE.ode4ms(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4s
+        ts,timeseries_tmp = ODE.ode4s(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
     end
```

Comment:
>Again, if you're willing to give up the type declaration

No, that would be a huge change and would have some big downsides for usage with things like Sundials.jl.

I could also just do the call via `typeof(alg)` since now they are the same name. The problem is that not all of the algorithms do accept `kwargs` parameters (`ode4` is an example which does not). Maybe the better solution to make them all accept it?